viewer_server/client_db_access_server.py

```diff
@@ -20,39 +20,127 @@
 from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
 from SimpleHTTPServer import SimpleHTTPRequestHandler
 
+from thrift import Thrift
+from thrift.Thrift import TException
 from thrift.transport import TTransport
 from thrift.protocol import TJSONProtocol
 
+import shared
 from codeCheckerDBAccess import codeCheckerDBAccess
+from codeCheckerDBAccess import constants
+from codeCheckerDBAccess.ttypes import *
 
 from client_db_access_handler import ThriftRequestHandler
 
+from Authentication import codeCheckerAuthentication
+from Authentication import constants
+from Authentication.ttypes import *
+
+from client_auth_handler import ThriftAuthHandler
+
 from codechecker_lib import logger
 from codechecker_lib import database_handler
+from codechecker_lib import session_manager
+
+import base64
 
 LOG = logger.get_new_logger('DB ACCESS')
 
 
-class RequestHander(SimpleHTTPRequestHandler):
+class RequestHandler(SimpleHTTPRequestHandler):
     '''
     Handle thrift and browser requests
     Simply modified and extended version of SimpleHTTPRequestHandler
     '''
 
     def __init__(self, request, client_address, server):
-
         self.sc_session = server.sc_session
 
         self.db_version_info = server.db_version_info
+        self.manager = server.manager
 
         BaseHTTPRequestHandler.__init__(self,
                                         request,
                                         client_address,
                                         server)
 
-    def log_message(self, msg_format, *args):
-        ''' silenting http server '''
-        return
+    #def log_message(self, msg_format, *args):
```

Comment:
This was a debug change that was accidentally added in development. It has been reverted ever since.

> GitHub does not hide the comments for hunks that had been modified since...